qa/tasks/salt_manager.py

```diff
@@ -207,6 +208,32 @@ def master_rpm_q(self, pkg_name):
             installed = False
         return installed
 
+    def __get_rpm_pkg_version(self, pkg):
```

Comment:
please move this function to `qa/tasks/util.py` and call it `get_rpm_pkg_version`. It should take a `remote` argument instead of using `self.master_remote`.